src/core/a-entity.js

```diff
@@ -278,7 +278,11 @@ var proto = Object.create(ANode.prototype, {
   initComponent: {
     value: function (name, data, isDependency) {
       var component;
+      var componentInfo = name.split('__');
+      var id = componentInfo[1];
```

Comment:
it will return `undefined`, we could leave it that way or do something like `componentInfo[1] || null`. In any case it should works as expected on the rest of the code where the full name is built using: `componentName = id ? component + '__' + id : component` and both `undefined` or `null` will return false.